app/exports/common_columns/_common_columns.yml

```diff
@@ -89,3 +89,10 @@ rejection_reason:
   type: string
   description: The reason the provider gave for rejecting the candidate
   example: Absence of English GCSE
+
+structured_rejection_reasons:
+  type: string
+  description: A list of structured reasons for rejection, split by new lines (\n)
```

Comment:
I've seen a few different approaches to lists and it would be good to commonise these. But I think that is something to do as a separate piece of work.

Reply:
I fixed this as part of this PR in the end